types/net-keepalive/tslint.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-unnecessary-generics": false,
+        "semicolon": [true, "never"],
+        "eofline": false,
```

Comment:
I believe `eofline` and `no-unnecessary-generics` rules shouldn't be disabled.

Reply:
Contribution guide says

> At this point we do not enforce a specific general code style, but we do like idiomatic code in each individual file.

But I can agree with the `no-unnecessary-generics`, code still lints without with the rule enabled. But could you elaborate with `eofline` issue why it *should* be enabled?